src/nvim/cursor_modify.h

```diff
@@ -0,0 +1,9 @@
+#ifndef NVIM_CURSOR_MODIFY_H
+#define NVIM_CURSOR_MODIFY_H
+
+#include "nvim/vim.h"
+
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+# include "cursor_modify.h.generated.h"
+#endif
+#endif  // NVIM_CURSOR_MODIFY_H
```

Comment:
I know it's in the style guide's example, but a space between the endif statements helps readability IMO.